app/controllers/notifications_controller.rb

```diff
@@ -4,20 +4,24 @@ class NotificationsController < ApplicationController
   before_action :render_home_page_unless_authenticated, only: [:index]
 
   def index
-    scope = current_user.notifications
-    scope = params[:archive].present? ? scope.archived : scope.inbox
+    notifications = current_user.notifications
```

Comment:
That makes sense. I only changed the name because scope was being assigned twice in a row here without being read. Is there a name for the variable notifications that would make more sense to you?